example/index.js

```diff
@@ -3,7 +3,7 @@ const syncComplete = (res) => {
   updateProfileData(window.box)
 }
 
-Box.create(window.ethereum).then(box => {
+Box.create(window.ethereum, { iframeCache: true }).then(box => {
```

Comment:
Using this should be default. There should be an option to disable it.

Reply:
Gotcha